### DISCUSSION TITLE
In which language was "All 🔇 on the ⬅️💥" originally published?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#891] [Galaxy Trivia] [answered]
**Bonus:** What year was the novel first published in 🇬🇧?

**Comment by peterbaccaro** [comment, marked as answer]
The novel "All Quiet on the Western Front" was originally published in **German**. Its original title is "Im Westen nichts Neues," and it was written by German author Erich Maria Remarque. 

The first UK edition of the novel was published by Putnam in **1929**.
